### ISSUE TITLE
Validation set results

### ISSUE THREAD

**Issue by codeskings** [haranrk/DigiPathAI#15]
Hello

Thank you for providing this pipeline. I have a question about the validation set (10 WSIs) of the PAIP2019 dataset; in the supplementary data of your paper, you mentioned the clipped Jaccard index of the individual models that make up the ensemble (table 9). 
Using the pipeline, I produced the predictions of the validation set for the Densenet model, but when I calculated the clipped metric for the generated predictions it gave (0.595517) while in your supplementary file, you stated that it was (0.679). 
Can you please explain why I got that different results? Is the used pre-trained model in the pipeline the same as the one you used to get that results?

Many thanks
